examples/backends/s3/configuration_hcl_s3_custom_s3_backend.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: terraform.core.oam.dev/v1beta2
```

Comment:
Remove `hcl` from the name please as we don't support json any more.

Reply:
OK